test/distributed/test_dynamo_distributed.py

```diff
@@ -700,13 +700,13 @@ def test_fsdp_skip_guards(self):
         Note: comptime prints the guards before the time they get installed or not installed, so in both cases
         (skip or no skip) the same guards get printed.  The difference is that in the skip case, they show up
         with a special 'guard source' which will cuase them to not be installed.  So all we check for is the expected
-        guard source 'local_fsdp_module'.
+        guard source 'local_nn_module'.
```

Comment:
hm, this test is essentially broken now that we collapsed the guard source, right? not the end of the world, but if it is, maybe just delete it instead of keeping it passing for wrong reason

Reply:
Is it broken? To me, the purpose changed to ensuring fsdp guards are installed, rather than that the fsdp guard source is specific to fsdp.

Reply:
oh. actually, do we skip guards on local_nn_module? (so then, no behavior change and i'm wrong)